app/Http/Resources/V1/NotificationResource.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace App\Http\Resources\V1;
+
+use Illuminate\Http\Request;
+use Illuminate\Http\Resources\Json\JsonResource;
+
+class NotificationResource extends JsonResource
```

Comment:
Perhaps, call it NotificationListResource. (Yury might object to it if it were NotificationResource.)